plugins/inputs/aerospike/aerospike.go

```diff
@@ -65,67 +72,231 @@ func (a *Aerospike) gatherServer(hostport string, acc telegraf.Accumulator) erro
 		iport = 3000
 	}
 
-	c, err := as.NewClient(host, iport)
+	var c *as.Client
+	if urlInfo.User != nil {
+		userInfo := urlInfo.User
+		clientPolicy := as.NewClientPolicy()
+		clientPolicy.User = userInfo.Username()
+		clientPolicy.Password, _ = userInfo.Password()
```

Comment:
I have a question, Influxdb output plugin have only one cluster so one tls config works. Aerospike plugin supports multiple cluster. Should I modify it to take nodes of one cluster only.